revolv/project/utils.py

```diff
@@ -6,3 +10,31 @@ def get_solar_csv_url(csv_id, mode):
     url = "http://home.solarlog-web.net/sds/modul/SolarLogWeb/Statistik.php?logid=0&c="
     url += csv_id + "&mode=" + str(mode) + "&offset=0&flag=32&ex=csv"
     return url
+
+def get_statistic_for_user(user_profile, attr):
```

Comment:
I just realized it makes a lot of sense for this to be a method of RevolvUserProfile. What do you think?